src/main/resources/application.properties

```diff
@@ -12,3 +12,8 @@ server.error.include-message=always
 # Logging file path
 logging.file.path=logs
 auth.token.public.key=${PUBLIC_KEY}
+server.port=8082
+
+
+spring.data.mongodb.uri=mongodb://localhost:27017/notification-service?readPreference\=primary&appname\=MongoDB%20Compass&ssl\=false
```

Comment:
Don't hardcode the Mongo URI here. Instead take it externally as a VM Argument or the Environment Variable.